src/configs.js

```diff
@@ -54,13 +55,27 @@ function validateConfig({ username, accessToken, repositories }) {
 function getConfig() {
   try {
     const CONFIG_FILE_PATH = env.getConfigFilePath();
+    const stat = fs.statSync(CONFIG_FILE_PATH);
+    const hasGroupRead = stat.mode & fs.constants.S_IRGRP;
+    const hasOthersRead = stat.mode & fs.constants.S_IROTH;
+    if (hasGroupRead || hasOthersRead) {
+      const error = new Error(
+        `Config file at ${CONFIG_FILE_PATH} needs to have more restrictive permissions. Run the ` +
+          'following to limit access to the file to just your user account:\n' +
+          '\n' +
+          `  chmod 600 "${CONFIG_FILE_PATH}"\n`
+      );
```

Comment:
@spalger Now I merged this, but what do you think about running the command automatically, like
```js
fs.chmod(CONFIG_FILE_PATH, 0600)
```
?

Not sure how this would work on a windows machine though.

Reply:
Yeah, it won't work on Window and I'm kind of grateful that other tools have forced me to do it (or I wouldn't have known to enforce it here 😁)